src/pages/Categories.js

```diff
@@ -1,22 +1,43 @@
 import React, { Component } from "react";
-import ProductList from "../components/ProductList";
-import '../components/products-list.scss';
 
 class Categories extends Component {
```

Comment:
This is the page which uses the `prop`, `renderCategoriesListPage` which is passed from the layout file.